### DISCUSSION TITLE
Documentos para inscrição via nota de empenho para a Python Cerrado + Plone Conference

### DISCUSSION THREAD

**Discussion by anadulce** [apyb/comunidade#170] [Pergunta]
Boa tarde, pessoal! 

Sou eu de novo pedindo documentos pra nota de empenho, desculpa :pray: 

* Documentação dos sócios ( RG e CPF); - da Any eu já tenho
* Certidões negativas (Débitos Relativos aos tributos Federais e à Dívida Ativa da União, Improbidade Administrativa e CEIS);

Além disso, precisaria que os seguintes documentos fossem preenchidos em papel timbrado da APyB e assinados:
* https://docs.google.com/document/d/12_3qyG_Aoh0opEGVKpWzElFmG6NgW63j/edit?usp=sharing&ouid=105012949324751328317&rtpof=true&sd=true
* https://docs.google.com/document/d/1vs4e95r8bJ326nPtyyP4pKgymqUZx8uy/edit?usp=drive_link&ouid=105012949324751328317&rtpof=true&sd=true

@apyb/diretoria

**Comment by anicelysantos** [comment]
Blz, @anadulce. Te retorno em breve

**Comment by anicelysantos** [comment]
Aqui @anadulce todos os docs solicitados: https://drive.google.com/drive/folders/1XOS9S--wQdl_wUeltnpbdrwZVCSDMABu?usp=drive_link
Ali no CEIS ele gerou um doc resumo, dá uma olhada se resolve.
